yt/geometry/grid_geometry_handler.py

```diff
@@ -248,7 +248,7 @@ def _find_field_values_at_points(self, fields, coords):
         coords = self.ds.arr(ensure_numpy_array(coords), "code_length")
         grids = self._find_points(coords[:, 0], coords[:, 1], coords[:, 2])[0]
         fields = list(iter_fields(fields))
-        mark = np.zeros(3, dtype=np.int)
+        mark = np.zeros(3, dtype=int)
```

Comment:
```suggestion
        mark = np.zeros(3, dtype="int64")
```
I think below we use int64, so we should make it the same here to be explicit.